org.gaphor.Gaphor.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.gaphor.Gaphor.desktop</id>
+  <name>Gaphor</name>
+  <summary>Simple modeling tool</summary>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
You have a duplicated metadata license here.

Reply:
Whoops, thanks for the thorough review, fixed!